test/index.js

```diff
@@ -1,8 +1,36 @@
-const expect = require('chai').expect,
+const expect = require('chai').expect;
 const toRGBAArray = require('..');
+const Canvas = require('canvas');
+const range = require('lodash.range');
+const fs = require('fs');
+const path = require('path');
 
-describe('my thing', function () {
-  it('should work', function () {
-    toRGBAArray('thing');
+describe('to-rgba-array', function () {
+  it('will work with a canvas', function () {
+    var canvas = new Canvas(200, 200);
+    return toRGBAArray(canvas).then(function (result) {
+      expect(result).to.be.a('Uint8ClampedArray');
```

Comment:
Or just use a very small canvas (eg. 16x16) and include the whole Buffer for comparison.

Also good to check something with an alpha channel to and check the alpha bytes.